src/Compilers/Core/Portable/InternalUtilities/OneOrMany.cs

```diff
@@ -250,14 +297,35 @@ public bool MoveNext()
 
             public T Current => _collection[_index];
         }
+
+        private sealed class DebuggerProxy(OneOrMany<T> instance)
+        {
+            [DebuggerBrowsable(DebuggerBrowsableState.RootHidden)]
+            public T[] Items => instance.ToArray();
+        }
+
+        private string GetDebuggerDisplay()
+            => "Count = " + Count;
     }
 
     internal static class OneOrMany
     {
         public static OneOrMany<T> Create<T>(T one)
             => new OneOrMany<T>(one);
 
+        public static OneOrMany<T> Create<T>(T one, T two)
+            => new OneOrMany<T>(ImmutableArray.Create(one, two));
+
+        public static OneOrMany<T> OneOrNone<T>(T? one)
+            => one is null ? OneOrMany<T>.Empty : new OneOrMany<T>(one);
```

Comment:
This might be quite surprising. If I'm calling such API with `null`, I'd expect the outcome to be a `OneOrMany` that has `Count == 1`, and `oneOrMany[0]` being null.

Returning Empty on null might not be clear without looking at the implementation.